fluentforms/core/src/main/java/com/_4point/aem/fluentforms/testing/assembler/MockTraditionalAssemblerService.java

```diff
@@ -17,17 +16,24 @@
 import com.adobe.fd.assembler.client.PDFAValidationResult;
 
 public class MockTraditionalAssemblerService implements TraditionalDocAssemblerService {
-	
+
 	private final AssemblerResult DUMMY_ASSEMBLER_RESULT;
-	
-    AssemblerResult assemblerResult;
-    GenerateAssemblerResultArgs generateAssemblerResultArgs ;
-    
-   
+
+	public AssemblerResult getDUMMY_ASSEMBLER_RESULT() {
+		return DUMMY_ASSEMBLER_RESULT;
+	}
+
+	AssemblerResult assemblerResult;
+	GenerateAssemblerResultArgs generateAssemblerResultArgs;
+
+	public GenerateAssemblerResultArgs getGenerateAssemblerResultArgs() {
+		return generateAssemblerResultArgs;
+	}
+
 	@Override
 	public AssemblerResult invoke(Document ddx, Map<String, Object> inputs,
 			AssemblerOptionsSpec adobAssemblerOptionSpec) throws AssemblerServiceException, OperationException {
-		this.generateAssemblerResultArgs = new GenerateAssemblerResultArgs(ddx, inputs,adobAssemblerOptionSpec);
+		this.generateAssemblerResultArgs = new GenerateAssemblerResultArgs(ddx, inputs, adobAssemblerOptionSpec);
 		return this.assemblerResult == null ? DUMMY_ASSEMBLER_RESULT : this.assemblerResult;
```

Comment:
Only one of the code paths on this statement is tested in the unit tests (where the user has provided an assemblerResult ahead of time).  It would be nice of both were tested.  This should be as easy as duplicating the test you have already created but removing the line where the assemblerResult is set.